ngmix/joint_prior.py

```diff
@@ -860,1514 +972,3 @@ def fill_fdiff(self, pars, fdiff, **keys):
         fdiff[0:index] = sqrt(chi2)
 
         return index
-
```

Comment:
Are there any class here that we need to document as removed in the changelog?

Reply:
No, but there is a new one I will put in the new features